Add documentation regarding limatations on Istio, Ory and API Gateway (#17249)
kyma-project/kyma: docs/01-overview/main-areas/api-exposure/apix-03-api-gateway-limitations.md

---
title: API Gateway limitations
---

## Controller limitations

The API Rule controller is not a critical component of Kyma networking infrastructure, since it relies on Istio and Ory Custom Resources to provide routing capabilities. In terms of persistence the controller depends on API Rules stored in the Kubernetes cluster and no other persistence solution is present.

> In terms of resource configuration set on API Gateway controller they are as follows:

Reviewer: ```suggestion
In terms of the resource configuration, the following limitations are set on the API Gateway controller:
```
